config/routes.rb

```diff
@@ -20,6 +20,8 @@
         get '/by_name', to: 'by_name#index'
       end
 
+      # get '/mentor_ship/users', to: 'mentorship#index'
```

Comment:
Why is this route commented out? Are you using it or just using the /users endpoint on line 27?